HWs/Test3/task1/main.c

```diff
@@ -0,0 +1,126 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdlib.h>
+
+#define binarySize 256
```

Comment:
```suggestion
#define BINARY_SIZE 256
```
Вот так нужно define константы объявлять
(-0.1)